### PR TITLE
Fix serious mistake

### PR DESCRIPTION
**Pull request by elliottzheng**
**Hi, you made a serious mistake in your training and testing code，you take the ground truth as the input of the generator, rather than the masked ground truth, which leads to the following situation. It means your experiment is completely wrong, maybe you should do them again.**
image
**I modified the code based on intuition, please check the code before merge.**

### CODE REVIEW

**Comment by Arbiter0** [comment]
@elliottzheng Hi! Did you complete this experiment? According to the modified code, I found that G and D could not form a confrontation, and D_loss was always a fixed value. As shown in the following figure
image
image

**Comment by elliottzheng** [comment]
@Arbiter0 Hi, it has been almost a year, I don't remember the detail, but I think I just follow the idea of the code and make it right. if the idea is wrong, then I guess it would;t work.

**Comment by Arbiter0** [comment]
@elliottzheng OK! Thanks for your reply!

**Comment by Arbiter0** [comment]
@elliottzheng Hi! I tested the model and got a result, but the result is vague. Do you hava any tricks to improve it? Thanks! I trained the model with `epoch=100` and the number of training data is 6000.
myplot

**Review by 7AM7 — APPROVED** [review]
